docs/content/concepts/configuration/config-schema-legacy.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: Run Configuration (Legacy) | Dagster
+description: Job run configuration allows providing parameters to jobs at the time they're executed.
+---
+
+# Run Configuration (Legacy)
+
+<Note>
+  This guide covers using the legacy Dagster config system. For docs on the new
```

Comment:
I would not describe this as "legacy Dagster config system" but instead just legacy APIs for the config system